src/components/SignUp/SignUp.js

```diff
@@ -156,198 +159,202 @@ export const SignUp = () => {
             progress: undefined,
             theme: "light",
           });
+          setLoad(false);
         }
-
-      } else {
-        toast.error("Datos inválidos", {
-          position: "top-center",
-          autoClose: 4000,
-          hideProgressBar: false,
-          closeOnClick: true,
-          pauseOnHover: false,
-          draggable: true,
-          progress: undefined,
-          theme: "light"
-        });
+      }, 3000);
+    } else {
+      toast.error("Datos inválidos", {
+        position: "top-center",
+        autoClose: 4000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: false,
+        draggable: true,
+        progress: undefined,
+        theme: "light",
+      });
     }
-}
-
-  return (
-    <Body>
-      <Container>
-        <ToastContainer
-          closeButton={true}
-          position="bottom-right"
-          autoClose="3000"
-          hideProgressBar="true"
-        />
-        <Link to="/">
-          <ContainerHeader>
-            <ContainerImage
-              src={require("../../assets/image-logo.png")}
-              width="70"
-              height="70"
-              alt="Logotipo Empresa"
-            />
-            <LabelHeader >REGISTRO</LabelHeader>
-          </ContainerHeader>
-        </Link>
+  };
+  if (load === false) {
+    return (
+      <Body>
+        <Container>
+          <ToastContainer
+            closeButton={true}
+            position="bottom-right"
+            autoClose="3000"
+            hideProgressBar="true"
+          />
+          <Link to="/">
+            <ContainerHeader>
+              <ContainerImage
+                src={require("../../assets/image-logo.png")}
+                width="70"
+                height="70"
+                alt="Logotipo Empresa"
+              />
```

Comment:
i think this you could change it to this 
<img width="937" alt="image" src="https://user-images.githubusercontent.com/76448998/214905811-c7e91bc1-a4ce-4119-864c-89b0c5d21f99.png">

looks a little more clear 

```javascript
return load ? <Loader /> : <SignUpForm />
```

and create a component only for the form and use props to comunicate to the parent component